lib/index.js

```diff
@@ -129,6 +129,48 @@ async function installSlipstreamCLI(downloadURL) {
   core.endGroup();
 }
 
+async function installAWSCLI() {
+  const commandExistsAlready = await commandExists('aws');
+
+  if (commandExistsAlready) {
+    return;
+  }
+
+  core.startGroup('Installing the AWS CLI');
```

Comment:
Good call, the `installAWSCLI` function should not install the aws cli if it exists already on the runnuer so it should not be an issue to keep it this way. But I could also just remove it, if it is installed on `every` runners now, I ask that because you mentioned it runs on `most` runners :)

Reply:
😄 I did not check all the runner which is the reason why I said most of them. I'm pretty sure it's available on all runners (just checked all the linux distributions). I would prefer to remove it. The smaller that Action is, the easier it will be to reason about in the future.

Reply:
I did check yesterday, and it worked just fine. I will remove it 👍